Cargo.toml

```diff
@@ -24,8 +24,8 @@ include = [
 dbus = { version = "0.9", optional = true }
 lazy_static = { version = "1", optional = true }
 image = { version = "0.24", optional = true }
-zvariant = { version = "3.2", optional = true }
-zvariant_derive = { version = "3.2", optional = true }
+zvariant = { version = "3.7", optional = true }
+zvariant_derive = { version = "3.7", optional = true }
```

Comment:
@flukejones Hey. Glad to see you doing the great work here. :) Is the direct dependency on zvariant crates really needed here btw? It's best to use it through the zbus re-export. Also, `zvariant_derive` should almost never be used directly. The macros are re-exported by `zvariant`.